python/stempy/image/__init__.py

```diff
@@ -1,9 +1,40 @@
 from stempy import _image
+from stempy import _io
 import numpy as np
-
-def create_stem_images(reader, inner_radii,
+import h5py
+
+
+def get_hdf5_reader(h5file):
+    # the initialization is at the io.cpp
+    dset_frame=h5file['frames']
+    dset_frame_shape=dset_frame.shape
+    totalImgNum=dset_frame_shape[0]
+    imagewidth=dset_frame_shape[1]
+    imageheight=dset_frame_shape[2]
+    
+    dset_stem_shape=h5file['stem/images'].shape
+    scanwidth=dset_stem_shape[1]
+    scanheight=dset_stem_shape[2]
+    
+    # the number here are same with the smallscan data
+    blocksize=32
+    blockNumInFile=25
```

Comment:
If these are constants we should move them to the top of the file and use uppercase.